packages/react/package.json

```diff
@@ -119,7 +119,8 @@
     "react-popper": "2.2.3",
     "react-spring": "9.3.0",
     "react-use-measure": "2.0.1",
-    "react-virtual": "2.2.7"
+    "react-virtual": "2.2.7",
+    "typescript": "4.5.5"
```

Comment:
Had to move typescript to dependencies, otherwise there were many type errors regarding various libraries, including popper